engine/server/engine/centralized_logs/client_implementations/kurtosis_backend/kurtosis_backend_logs_database_client.go

```diff
@@ -192,7 +193,8 @@ func streamServiceLogLines(
 				return
 			}
 
-			logLine := logline.NewLogLine(logLineStr)
+			// TODO: Parse out the timestamp from docker container logs string instead of time.Now
+			logLine := logline.NewLogLine(logLineStr, time.Now())
```

Comment:
if I was being nitpicking, I would say we should move time.Now() up to being as close to when the line is received. It should be the first thing we do once the new line is ready. In this case it probably won't change the timestamp much but if you had more time consuming logic above this particular line, the timestamp would be later than when we actually received it.

Reply:
Completely agree with ya. FWIW - right now we're not using the `KurtosisBackendLogsDatabaseClient` currently, but I wanted to implement timestamps for it to keep parity across the implementations of `LogsDatabaseClient`s

Reply:
For the `PersistentVolumeLogsDatabaseClient` (the implementation we use currently), we use the time that the `LogsAggregator` claims to have seen the logs, which semantically I'm considering as the equivalent of when Kurtosis sees the log.